Sources/Paywalls/Components/PaywallPurchaseButtonComponent.swift

```diff
@@ -16,6 +16,7 @@ public extension PaywallComponent {
         public let stack: PaywallComponent.StackComponent
 
         public let action: Action?
```

Comment:
Should we mark it as `deprecated`? It's actually unused (literally, if you apply my nit above)
```suggestion
        @available(*, deprecated, message: "This API is unused and will be removed in a future version.")
        public let action: Action?
```

Reply:
Can't do this deprecation because the `pod lib lint` job will yell at us 😅